h2o-py/tests/testdir_misc/pyunit_export_parquet_npe.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import tempfile
+
+sys.path.insert(1, "../../../")
+import h2o
+from tests import pyunit_utils
+
+
+def test_export_file_npe_gh_16161():
+    with tempfile.TemporaryDirectory() as dir:
+        df = h2o.create_frame(rows=100, cols=10, string_fraction=0.1, seed=5, seed_for_column_types=25)
+        h2o.export_file(df, path=dir, format="parquet", write_checksum=False)
+
```

Comment:
can you import the frame back after you export it and then compare with the original frame df to make sure they still equal?

Reply:
Done in the R test